plugins/scaffolder/src/components/Router.tsx

```diff
@@ -53,26 +50,18 @@ export type RouterProps = {
   }>;
 };
 
+/**
+ * The main entirypoint `Router` for the `ScaffolderPlugin`.
```

Comment:
```suggestion
 * The main entrypoint `Router` for the `ScaffolderPlugin`.
```

Signed-off-by: Fredrik Adelöw <freben@gmail.com>